fem/ceed/interface/coefficient.hpp

```diff
@@ -22,6 +21,8 @@
 #include "util.hpp"
 #include "ceed.hpp"
 
+#ifdef MFEM_USE_CEED
```

Comment:
Is there a reason this isn't above the include statements?

Reply:
Good catch, done in [c11576a](https://github.com/mfem/mfem/pull/2953/commits/c11576a686e107a11e97d5430ac22aa817c3b63b).